src/libraries/System.Security.Permissions/src/System/DirectoryServices/DirectoryServicesPermission.cs

```diff
@@ -5,7 +5,9 @@
 
 namespace System.DirectoryServices
 {
+#if NETCOREAPP
```

Comment:
Why the `#if`?  Shouldn't these remain obsolete for the `netstandard2.0` build?

Reply:
Thanks. Added those back; this required use of `<IncludeInternalObsoleteAttribute>true` in the csproj to avoid compile errors like `error CS0246: The type or namespace name 'DiagnosticId' could not be found `.